test/common.js

```diff
@@ -453,3 +453,9 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+exports.busyLoop = function busyLoop(time) {
+  var startTime = Date.now();
+  var stopTime =  startTime + time;
+  while (Date.now() < stopTime);
```

Comment:
I am on mobile right now. I would like to check if this will be an infinite loop if `stopTime` is `NaN`